README.md

```diff
@@ -96,4 +96,4 @@ Give a ⭐️ if this project helped you!
 [conan-recipe-repo]: https://github.com/eclipse-uprotocol/up-conan-recipes
 [spec-repo]: https://github.com/eclipse-uprotocol/up-spec
 [conan-abi-docs]: https://docs.conan.io/en/1.60/howtos/manage_gcc_abi.html
-[conan-txt-reference]: https://docs.conan.io/2/reference/conanfile_txt.html
\ No newline at end of file
+[conan-txt-reference]: https://docs.conan.io/2/reference/conanfile_txt.html
```

Comment:
It seems that various contributors' editors have different opinions about newlines at the end of files. We should probably figure out if we can standardize this somehow (or add a lint check) to prevent unnecessary churn.

Reply:
Agreed, I'd personally prefer a lint check just to help standardize it and make it uniform.